view/src/main/java/cmm/apps/esmorga/view/eventdetails/EventDetailsView.kt

```diff
@@ -44,7 +43,12 @@ import org.koin.core.parameter.parametersOf
 
 @Screen
 @Composable
-fun EventDetailsScreen(eventId: String, onBackPressed: () -> Unit, edvm: EventDetailsViewModel = koinViewModel(parameters = { parametersOf(eventId) })) {
+fun EventDetailsScreen(
+    eventId: String,
+    onBackPressed: () -> Unit,
+    edvm: EventDetailsViewModel = koinViewModel(parameters = { parametersOf(eventId) }),
+    onLoginClicked: () -> Unit
```

Comment:
would be nice to have this ordered, having it shorted like this, a callback, VM, another callback is quite messy.

Reply:
you're right ... I will do it